content/minfraud/api-documentation/responses.mdx

```diff
@@ -0,0 +1,229 @@
+        this service. Please [purchase more service credits](https://www.maxmind.com/en/purchase-minfraud-services).
+      </td>
+    </tr>
+    <tr>
+      <td>`PERMISSION_REQUIRED`</td>
+      <td>402 Payment Required</td>
+      <td>
+        You do not have permission to use the service. Please contact
+        support@maxmind.com for more information.
+      </td>
+    </tr>
+    <tr>
+      <td>(none)</td>
+      <td>403 Forbidden</td>
+      <td>
+        This status is returned when the request body is larger than 20,000
+        bytes.
+      </td>
+    </tr>
+    <tr>
+      <td>(none)</td>
+      <td>415 Unsupported Media Type</td>
+      <td>
+        Your request included a `Content-Type` header that is not supported. For
+        `GET` requests, this means the web service cannot return content of that
+        type. For `PUT` and `POST` queries, this means the web service cannot
+        parse a request body of that type.
+      </td>
+    </tr>
+    <tr>
+      <td>(none)</td>
+      <td>503 Service Not Available</td>
+      <td>
+        There is a problem with the web service server. You can try this request
+        again later.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Object Reference
+
+Each schema definition contains a description of an object, along with a list of
+properties that belong to the object. The following information is listed for
+each object property:
+
+  {/* <!-- lint disable list-item-bullet-indent --> */}
+  * name
+  * type (`array<type>`, `boolean`, `number`, `integer`, `object`, `string`)
+  * description
+  * example
+  * formatting
+  * constraints
+  * supported services (`Score`, `Factors`, `Insights`)
+  {/* <!-- lint enable list-item-bullet-indent --> */}
+
+Additionally, for `object` properties, a link is provided to view a schema
+definition that further describes that specific object.
+
+<Alert type="info">
+* Keys with undefined or empty values will not be included in the returned
+  in the response object.
+
+* The data returned in the document will be in UTF-8 encoding.
+
+* A given key and value may be omitted from the response entirely if there
+  is no relevant information to include. For example, if you do not pass any
+  information about the credit card in your request, then the response will
+  not contain a `credit_card` key or value.
+</Alert>
+
+<Schemas.Response />
+<Schemas.ResponseIpAddress />
+<Schemas.ResponseIpAddressCountry />
+<Schemas.ResponseIpAddressLocation />
+<Schemas.ResponseIpAddressRiskReasons />
+<Schemas.ResponseIpAddressRiskReason />
+<Schemas.ResponseCreditCard />
+<Schemas.ResponseCreditCardIssuer />
+<Schemas.ResponseDevice />
+<Schemas.ResponseEmail />
+<Schemas.ResponseEmailDomain />
+<Schemas.ResponseShippingAddress />
+<Schemas.ResponseBillingAddress />
+<Schemas.ResponseDisposition />
+<Schemas.ResponseSubscores />
+<Schemas.ResponseWarnings />
+<Schemas.ResponseWarning />
```

Comment:
At the bottom of the current page there are two sections, Versioning and License https://dev.maxmind.com/minfraud/#Versioning and https://dev.maxmind.com/minfraud/#License

I don't see that text anywhere on the new site, it seems like it could be added to the left nav under the API Documentation section as a separate section from the Requests and Responses. Open to suggestions

Reply:
Those can be found on the minFraud API Documentation overview page - https://dev-maxmind-com--pr123-wes-mf-response-ldjkqvog.web.app/minfraud/api-documentation

Reply:
gotcha, looks good.